Dockerfile-alpine.template

```diff
@@ -33,15 +33,10 @@ RUN addgroup -g 1000 node \
         libgcc \
         linux-headers \
         make \
+        bash \
         ${PYTHON_VERSION} \
     # gpg keys listed at https://github.com/nodejs/node#release-keys
-    && for key in \
-      "${NODE_KEYS[@]}"
-    ; do \
-      gpg --batch --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys "$key" || \
-      gpg --batch --keyserver hkp://ipv4.pool.sks-keyservers.net --recv-keys "$key" || \
-      gpg --batch --keyserver hkp://pgp.mit.edu:80 --recv-keys "$key" ; \
-    done \
+    && curl -s https://raw.githubusercontent.com/nodejs/release-keys/HEAD/cli.sh | bash -s import \
```

Comment:
Should `HEAD` instead be pinned to a commit has in the `versions.json` for stability? The CLI itself currently just pulls keys from the default branch, so I'm not sure if it buys much